groups/data-lake/variables.tf

```diff
@@ -1,22 +1,8 @@
-variable "region" {}
-
-variable "database_password" {
-  type        = string
-  description = "The Redshift database password"
-}
-
-variable "database_username" {
-  type        = string
-  description = "The Redshift database username"
-}
-
-variable "mongo_export_db_url" {
-  type        = string
-  description = "The MongoDB URL for the export Lambda"
+variable "aws_profile" {
+  default = "development-eu-west-2"
+  type = string
 }
 
-variable "mongo_export_s3_path" {
-  type        = string
-  default     = "DMS-Output/lambda"
-  description = "The S3 bucket prefix for the export Lambda; will be prefixed with the bucket name automatically"
+variable "region" {
+    type = string
```

Comment:
As above, can we add a suitable description here? For example, this is used in other projects:

```
description = "The AWS region in which resources will be created"
```